### DISCUSSION TITLE
logging-operator vs fluent Operator

### DISCUSSION THREAD

**Discussion by wangtao208208** [fluent/fluent-operator#1341] [General]
how about logging-operator vs fluent Operator ?   what should i choose from it

**Comment by benjaminhuo** [comment]
There are 4 combinations when using Fluent Operator:
- Using FluentBit alone as DaemonSet
- Using FluentBit alone as StatefulSet
- Using Fluentd alone as DaemonSet
- FluentBit DaemonSet -> Fluentd Statefulset

As far as I know there is only 1 mode in Logging Operator:
- FluentBit DaemonSet -> Fluentd Statefulset
